sources/include/citygml/citygml.h

```diff
@@ -69,6 +69,7 @@ namespace citygml
             , minLOD( 0 )
             , maxLOD( 4 )
             , optimize( false )
```

Comment:
as the current behavior was to run the tessellation by default, an initialization of the flag to false may break things for user. default initialization to true might be better